src/Symfony/Component/Amqp/bin/reset.php

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env php
```

Comment:
why not a command?
would need some explanations telling when this is useful.

Reply:
There are no dependency so I guess it's better to keep it as it. 
More over it's really an internal tool